### PR TITLE
Fix detect current user location + or browse list point to list on right.

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

- [x] Using open street maps api

cc @DavidLemayian 

Screen Shot 2019-09-24 at 7 50 52 AM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/lib/api.js [hunk 1]
```diff
@@ -1,16 +1,15 @@
 import axios from 'axios';
 import config from '../config';
 
-const GOOGLE_GEOCODE_URL =
-  'https://maps.googleapis.com/maps/api/geocode/json?sensor=false&language=en';
```
Comment (DavidLemayian) [inline src/lib/api.js]: Would see this as remaining here instead of in config coz not something you'd change for a new instance.

Comment (karimkawambwa) [reply src/lib/api.js]: Someone in the config could add some query options @DavidLemayian

**Comment by karimkawambwa** [comment]
@kilemensi or @DavidLemayian I will need that key for this PR

**Comment by karimkawambwa** [comment]
@KhadijaMahanga the key is still not valid, I think google disabled it. We need a new one from @DavidLemayian and this time I will have to most likely encrypt it before pushing and then decrypt it when it wants to be used because even on gh-pages I think it will be detected as an exposed key me thinks @kilemensi

File: src/lib/api.js (hunk 1 above)
Comment (DavidLemayian) [reply src/lib/api.js]: @karimkawambwa I see the query options happening here where e.g we automatically pass TLD for the country biasing - https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes

Also language I'd see as being dynamic to change according to user's preference of viewing the site.

File: src/components/Modal/PortalChooser.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   MenuItem,
   Button
 } from '@material-ui/core';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
```
Comment (DavidLemayian) [inline src/components/Modal/PortalChooser.js]: Are we able to try use the same arrow as the "Find Place" component? Not sure if it's SVG though to allow us play with opacity.

image

**Comment by karimkawambwa** [comment]
@kilemensi 
Screen Shot 2019-09-24 at 9 03 35 AM

**Pull request merged**